src/app/services/reducers/adaptive-cards-reducer.ts

```diff
@@ -0,0 +1,29 @@
+import { IAction } from '../../../types/action';
```

Comment:
Indentation is off in this file. It uses 4 instead of 2 spaces.

Reply:
Well spotted sir!